packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -265,4 +269,30 @@ export class LoginPage {
 			this.page.click( selectors.magicLinkContinueLoginButton ),
 		] );
 	}
+
+	/* 2FA */
+
+	/**
+	 * Enters the provided 2FA code and submits the form.
+	 *
+	 * @param {string} code Generated TOTP code.
+	 */
+	async enter2FACode( code: string ): Promise< void > {
+		await this.page.fill( selectors.mfaInput, code );
+
+		const [ response ] = await Promise.all( [
+			this.page.waitForResponse( '**/wp-login.php?action=two-step-authentication-endpoint' ),
```

Comment:
This endpoint is slightly different from the one used in `LoginPage.login`, but the overall outline of the method is the same.

In a subsequent PR (not this one due to scope creep) I think it's worth looking at ways to extract the commonalities into a separate method.